contribute/labels-of-issues.md

```diff
@@ -0,0 +1,116 @@
+
+---
+
+## Area
+
+Labels prefixed with *area* indicates that which area issue or PR belong. Here, area implies component of the OpenEBS project. It will be easier to sort an issue based on the area of user's interest.
+
+| Label | Description |
+|---|---|
+| area/networking  | Networking Specific  |
+| area/k8s | Kubernetes specific   |
+| area/ci  | Continuous integration specific  |
+| area/high-availability  | High availability  |
+| area/storage | Storage specific  |
+| area/machine-learning  |  Machine learning specific |
+
+---
+
+## Size
+
+Labels prefixed with *size* defines the size of change done in particular PR.
+
+| Label | Description |
+|---|---|
+| size/L | Size of change is large/7 commits probably |
+| size/M | Size of change is medium/5 commits probably |
+| size/S | Size of change is small/3 commits probably |
+| size/XL | Size of change is extra large/10 commits probably |
+| size/XS | Size of change is extra small/1 commit probably |
+| size/XXL | Size of change is very large/15 commits probably |
+
+### NOTE
+
+Please do not club major changes into a single PR. It'll be difficult for a reviewer to review a big PR. In that case, reviewer may discard your PR and will ask you to create multiple PRs.
+
+---
+
+### Priority
+
+Labels with prefix *priority* defines the severity of the issue. Severity level of an issue can be given as a suffix.
+
+| Label | Description |
+|---|---|
+| priority/backlog | Higher priority than priority/awaiting-more-evidence. |
+| priority/critical-urgent | Highest priority. Must be actively worked on as someone's top priority right now. |
+| priority/important-longterm | Important over the long term, but may not be staffed and/or may need multiple releases to complete. |
+| priority/important-soon | Must be staffed and worked on either currently, or very soon, ideally in time for the next release. |
+| priority/awaiting-more-evidence | Lowest priority. Possibly useful, but not yet enough support to actually get it done. |
+
+---
+
+### Sub-Project
+
+Labels with prefix *repo* points the subproject where given issue needs to be fixed. If users have no idea where to file the issue then they can file a issue in `openebs/openebs` later OpenEBS contributors/authors/owners will tag issue with the appropriate repository using *repo* label.
+
+| Label | Description |
+|---|---|
+| subproject/mayastor | Related to Replicated PV Mayastor and its components |
+| subproject/zfs-localpv | Related to ZFS Local PV and its components |
+| subproject/lvm-localpv | Related to LVM Local PV and its components |
+| subproject/dyamic-localpv |  Related to Hostpath Local PV and its components |
+| subproject/rawfile-localpv|  Related to Rawfile Local PV and its components |
+
+---
+
+### General
+
+As the name suggests, this contains general category labels and they are self-explanatory.
+
+| Label | Description |
+|---|---|
+| architecting | If something needs design changes/brainstorming |
```

Comment:
Update as per the org labels now please

Reply:
There are a lot of them, I think we should first trim down a bit? Not sure which ones to put here. Also I see a lot of labels in org which are not listed.

Reply:
Let's leave out the label discussion out of this PR. I will raise a different one once we all decide.